README.md

```diff
@@ -17,6 +17,11 @@ Currently, it mainly supports related work for AWS, especially [aws-account-oper
 - Go >= 1.13
 - make
 - [goreleaser](https://github.com/goreleaser)
+- `GOPROXY` contains `proxy.golang.org`
+  ```shell
+  # Google`s default proxy can be added globally:
+  go env -w GOPROXY="$(go env GOPROXY),https://proxy.golang.org"
```

Comment:
Indeed, on a branch new fedora system on which go is installed:
```
podman run -ti fedora:latest 
Resolved "fedora" as an alias (/etc/containers/registries.conf.d/000-shortnames.conf)
Trying to pull registry.fedoraproject.org/fedora:latest...
Getting image source signatures
Copying blob sha256:62946078034b7fe37984579d9b82ccf20cc98ffcd6517cf79ffad18e06fe2b23
Copying config sha256:2ecb6df959942dd2fdeb65606ca2e42a54f8c06af10eeb594fdfc3e2656c53d1
Writing manifest to image destination
Storing signatures
[root@d096e25a2678 /]# sudo dnf install golang
Fedora 36 - x86_64                                                                                                                                                                                                          8.7 MB/s |  81 MB     00:09    
Fedora 36 openh264 (From Cisco) - x86_64                                                                                                                                                                                    1.8 kB/s | 2.5 kB     00:01    
Fedora Modular 36 - x86_64                                                                                                                                                                                                  2.4 MB/s | 2.4 MB     00:00    
Fedora 36 - x86_64 - Updates                                                                                                                                                                                                 14 MB/s |  27 MB     00:01    
Fedora Modular 36 - x86_64 - Updates
...
```
We can see:
```
[root@d096e25a2678 /]# go env GOPROXY
direct
```

That said, I did the same for ubuntu... and `GOPROXY` was correctly filled with `https://proxy.golang.org` by default. From memory (cannot start an OS X container), the variable is also correctly initialized for Max OS.

So I guess this is a pb specific to some Linux distribs / some package managers, including Fedora.